pythclient/pythaccounts.py

```diff
@@ -501,6 +502,18 @@ def aggregate_price_confidence_interval(self) -> Optional[float]:
         """the aggregate price confidence interval"""
         return self.aggregate_price_info and self.aggregate_price_info.confidence_interval
 
+    async def get_aggregate_price_status(self, commitment: str = SolanaCommitment.CONFIRMED) -> Optional[PythPriceStatus]:
```

Comment:
I agree, but on the other hand it'll become a bit harder for people to use it.
I might create another normal (not async) function for it.

Reply:
i think it would be reasonable to have a non-async version where the caller passes in the slot, and also an async version that calls the non-async version.

Reply:
I removed async entirely as I realized using the provided slot when object is created is more efficient.